src/FSharp.AWS.DynamoDB/Expression/ConditionalExpr.fs

```diff
@@ -102,41 +100,42 @@ type QueryExpr with
 // 4. HashKey comparison must be equality comparison only.
 // 5. Must not contain OR and NOT clauses.
 // 6. Must not contain nested operands.
-let extractKeyCondition (qExpr : QueryExpr) =
+let extractKeyCondition (qExpr: QueryExpr) =
     let hashKeyRef = ref None
     let rangeKeyRef = ref None
+
     let trySet r v =
-        if Option.isSome !r then false
-        else r := Some v ; true
+        if Option.isSome !r then
```

Comment:
that rule needs to bs switched off

Reply:
I can’t find a setting for this one, there’s a line length setting for `if.. then.. else..` on one line, but nothing to let you split across 2

Reply:
Yes, looks like it reformats the whole expression to one line without it

Reply:
could be - maybe we should remove the ref cells and archaic syntax separately, but after formatting prob makes more sense

Reply:
maybe it makes sense then when thats in play
but see my mainline conclusion....

Reply:
If it’s important to keep the multiple-statements-on-one-line Eirik has used in the expression code, what I can do is ignore those and manually add back the relevant single-liners. Code changes in those files will then need to be formatted manually (although they don’t change much)

Reply:
no, this is fine - while semicolons and/or `in`s can be very expressive, peopelle will cope, and they are rare